Sources/MapboxNavigation/RoutesPresentationStyle.swift

```diff
@@ -18,4 +19,11 @@ public enum RoutesPresentationStyle {
      If `false` geometry of only first route will be used. Defaults to `true`.
      */
     case all(shouldFit: Bool = true)
+    
+    /**
+     All routes will be presented on a map view with the custom camera options.
+     
+     - parameter cameraOptions: The custom `CameraOptions` that the map view ill be presented with.
+     */
+    case custom(cameraOptions: CameraOptions)
```

Comment:
Do you think that it'd make sense to also pass a lit of routes here so that user has the ability to choose? Since it's public API I'd rather make this decision right now.

Reply:
Because the `RoutesPresentationStyle` is used in `navigationMapView.fitCamera(to:routesPresentationStyle:animated:)`, the custom routes or route options could be decided by the `routes` property instead. The `RoutesPresentationStyle` should works for custom `cameraOptions` only.